Framework/PythonInterface/plugins/algorithms/MaskShadowedPixels.py

```diff
@@ -0,0 +1,178 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
It seems to me that this algorithm is incomplete or abandoned half way through.
I think we should delete the file completely from this PR.

Reply:
@wdzhou Wenduo, I think at some point you touched algorithm `MaskShadowedPixels` but we're currently not using it in `_NOMADMedianDetectorTest`, `NOMADMedianDetectorTest`, or any of the tests. Can we remove the changes introduced and therefore take it away from this pull request?